lib/parse-args.js

```diff
@@ -126,7 +128,8 @@ function buildYargs (withCommands = false) {
     })
     .option('temp-directory', {
       describe: 'directory V8 coverage data is written to and read from',
-      default: process.env.NODE_V8_COVERAGE
+      default: process.env.NODE_V8_COVERAGE,
+      type: 'string'
     })
```

Comment:
I changed this line back to what it was in earlier versions.  This should address the UI prompts as indicated in the [PR comments.](https://github.com/bcoe/c8/pull/447#pullrequestreview-1265881416)